### PR TITLE
Add homebus image for eext and a bdr image

### PR DESCRIPTION
**Pull request by aajith-arista**
Add homebus image for eext and a bdr image

    1. Added a homebus image to be used for homebus creation from Aboard.
    2. Added a docker image to be pulled from
       barney-docker.infra.corp.arista.io.
        - Added an entrypoint to initialize the user in the container.
    3. Added README.md for users.

Note that this depends on work in:
https://github.com/aristanetworks/eos-external-tools/pull/116
https://github.com/aristanetworks/eos-external-tools/pull/118

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+1. eext docker container on existing EOS homebus.
+2. eext homebus.
+
+Both of them share most of the parts of the workflow. The choice you have to make is whether you want to work in your EOS home bus itself, or would you rather prefer having a separate homebus for your eext work.
+Once you enter the environment with the eext tool/binary, both workflows converge.
+Pls refer to the [eext User Guide](https://aid/13327) on how your work with the eext tool. This page just helps you setup the environment that has the eext tool avaiable.
+
+## eext docker container on existing EOS homebus
+Using [Barney Docker Registry](https://aid/11103), we export a docker image for the eext homebus development environment.
+Your EOS homebus and supports running docker containers. You can run an eext development container in either environment.
```
Comment (125priyank) [inline README.md:14]: nit: Your EOS homebus supports running ...

File: README.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
+# Interactive workflow for eext
+This repo hosts the code for supporting an interactive worfklow for [eext](https://aid/13327).
+
+There are two variants for this workflow:
+1. eext docker container on existing EOS homebus.
+2. eext homebus.
```
Comment (125priyank) [inline README.md:6]: Can you add a comment here too in brackets `(not yet supported)`.

File: docker/entrypoint.bash [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+if [[ ! -f "${INITIALIZED_FILE}" ]]; then
+   setup_user
+   setup_dirs
+   install_packages
+   touch "${INITIALIZED_FILE}"
+fi
+
+# Block indefinetely
+# We use docker exec for further usage
+while true; do sleep 10000; done
```
Comment (125priyank) [inline docker/entrypoint.bash:49]: Better would be to have `sleep infinity`

File: README.md (hunk 1 above)
Comment (aajith-arista) [reply README.md:14]: will fix

File: README.md (hunk 2 above)
Comment (aajith-arista) [reply README.md:6]: will fix

File: docker/entrypoint.bash (hunk 3 above)
Comment (aajith-arista) [reply docker/entrypoint.bash:49]: I tried, the rhel9 version of sleep doesn't support `infinity`. I think it's only there in alpine

File: README.md [hunk 4]
```diff
@@ -0,0 +1,91 @@
+# Interactive workflow for eext
+This repo hosts the code for supporting an interactive worfklow for [eext](https://aid/13327).
+
+There are two variants for this workflow:
+1. eext docker container on existing EOS homebus.
```
Comment (125priyank) [inline README.md:5]: I would suggest you to also add 
1. eext docker container on existing EOS homebus **(suggested workflow)**

File: barney.yaml [hunk 5]
```diff
@@ -1,9 +1,38 @@
...
+
+  homebus:
     units:
-      - image: .%bus
+      - image: code.arista.io/eos/tools/eext%base-image-devel
+      - image: code.arista.io/eos/tools/eext%go-binaries
+      - image: code.arista.io/eos/tools/eext%eext-configfiles
+    entry:
+      env:
+        EEXT_DEPSDIR: "/dest/RPMS"
```
Comment (125priyank) [inline barney.yaml:15]: To fix homebus create error `b5 failed to mount /var/run/dockersocket`.
add a finalizer
```
    finalizers:
      - [ rm, -f, /var/run ]
      - [ mkdir, /var/run, -m, "0755" ]
```

**Review by 125priyank — APPROVED** [review]

**Pull request merged**